notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    scheduler = UniPCMultistepScheduler.from_config(pipe.scheduler.config)
</pre><p>This <code>scheduler</code> is not used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
removed!